src/functionalTest/java/io/knotx/stack/functional/KnotxFragmentsDebugDataWithHandlebarsTest.java

```diff
@@ -94,6 +100,24 @@ void requestPage(VertxTestContext testContext, Vertx vertx,
             JsonObject nodeLog = new JsonObject(result);
             assertNotNull(nodeLog);
             assertEquals(8, nodeLog.getJsonObject("log").getJsonArray("operations").size());
+            JsonArray nodes = nodeLog.getJsonObject("log").getJsonArray("operations");
+            assertEquals("fetch-user-info", nodes.getJsonObject(0).getString("node"));
+
+            List<String> actions = new ArrayList<>(); //all actions which will occur in the composite node execution
+            actions.add("fetch-payment-providers");
+            actions.add("fetch-offers");
+            actions.add("fetch-offers-fallback");
+            actions.add("fetch-delivery-options-cb");
+            actions.add("fetch-delivery-timeout");
+
+            for (int i = 1; i < 6; i++) {
```

Comment:
Can you please make this more verbose:
`assertEqualsWithRange("fetch-payment-providers", nodes, 1, 6)`